Utilities/File.h

```diff
@@ -377,15 +377,25 @@ namespace fs
 		}
 
 		// Read std::basic_string
-		template <typename T>
-		std::enable_if_t<std::is_trivially_copyable_v<T> && !std::is_pointer_v<T>, bool> read(std::basic_string<T>& str, usz size,
+		template <bool is_strict = false, typename T>
```

Comment:
Please use CamelCase in template parameters

Reply:
fixed.